cmd/forwarder/credential.go

```diff
@@ -0,0 +1,6 @@
+package main
+
+type Credential struct {
```

Comment:
Godoc please

Reply:
@freeformz I have added godoc and made Credential private, moving it into the `auth.go` package. Should I *remove* the godoc now that it's private?

Reply:
I'd keep it. docs are good.